MAD.Integration.TableauCRM/Jobs/JobManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Hangfire;
+using MAD.Integration.TableauCRM.Data;
+
+namespace MAD.Integration.TableauCRM.Jobs
+{
+    public class JobManager
+    {
+        private readonly ConfigurationDbContext dbContext;        
+        private readonly IRecurringJobManager recurringJobManager;
+        private readonly IBackgroundJobClient backgroundJobClient;
+
+        public JobManager(ConfigurationDbContext dbContext, IRecurringJobManager recurringJobManager, IBackgroundJobClient backgroundJobClient)
+        {
+            this.dbContext = dbContext;            
+            this.recurringJobManager = recurringJobManager;
+            this.backgroundJobClient = backgroundJobClient;
+        }
+
+        public async Task CreateOrUpdateJobs() => await Task.Run(() =>
+        {
+            // Delete inactive jobs
+            foreach (var configuration in this.dbContext.Configuration.Where(y => y.IsActive == false))
```

Comment:
Instead of doing Task.Run, do var configs = await dbContext.Configuration.Where(..).ToListAsync()